SPM_Node.X/SPM.c

```diff
@@ -1,10 +1,167 @@
 #include "SPM.h"
 
+float analog_channels[36] = {0};
```

Comment:
I was thinking that 12 or 16 bits of precision wouldn't really matter for a double vs a float. But I guess it might in some cases.